### PR TITLE
Added card groups and card decks

### PR DESCRIPTION
**Pull request by Pallavi-PH**
Screenshot from 2020-03-04 13-02-57
Screenshot from 2020-03-04 13-03-09

### CODE REVIEW

**Comment by Pallavi-PH** [comment]
Screenshot from 2020-03-04 13-49-10

**Comment by Pallavi-PH** [comment]
Screenshot from 2020-03-04 13-58-55

File: scss/core/abstracts/_colors.scss [hunk 1]
```diff
@@ -358,3 +358,16 @@ $oranges: map-merge((
...
+$healthy-primary : $green-500;
+$healthy-secondary :$green-300;
+$unhealthy-primary : $red-500;
+$unhealthy-secondary : $red-300;
+$link-color : $blue-500;
+$btn-github:$gray-800;
+$code-wrap-bg:$gray-000;
+$active-code-block-shadow:$violet-400;
+$font-color-gray:$gray-700;
+$bg-g:rgba($gray-000, 0.5);
```
Comment (visualapps) [inline scss/core/abstracts/_colors.scss]: colors.scss only includes in the de-code color palette. It's should include other color mappings. Color variable for a comment should go in variable.scss

Comment (Pallavi-PH) [reply scss/core/abstracts/_colors.scss]: Done

**Pull request merged**